interoperability_report.py

```diff
@@ -427,49 +429,70 @@ def run_test(
     manager = Manager()
      # used for storing the obtained ReturnCode
      # (from Publisher and Subscriber)
-    code = manager.list(range(2))
+    return_code = manager.list(range(2))
     data = Queue() # used for storing the samples
 
     subscriber_finished = Event()
     publisher_finished = Event()
 
-
+    log_message('Creating temporary files', verbosity)
     file_publisher = tempfile.TemporaryFile(mode='w+t')
     file_subscriber = tempfile.TemporaryFile(mode='w+t')
 
+    # Manager is a share memory section where both processes can access.
+    # return_code is a list of two elements where the different processes
+    # (publisher and subscriber applications) will copy their ReturnCode.
+    # These ReturnCodes are identified by the index within the list,
+    # every index identifies one application. Therefore, only one application
+    # modifies one element of the list.
+    # Once both processes are finished, the list contains the ReturnCode
+    # in the corresponding index. This index is set manually and we need it
+    # in order to use it later.
+    # Example:
+    #   Processes:
+    #     - Publisher Process (produced_code_index = 1)
+    #     - Subscriber Process (produced_code_index = 0)
+    #   Code contains:
+    #     - return_code[1] contains Publisher Application ReturnCode
+    #     - return_code[0] contains Subscriber Application ReturnCode
```

Comment:
when we change this comment, we should also change similar comments in other parts of the script